client/package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "namex",
-  "version": "2.1.2",
+  "version": "2.1.3",
   "description": "BC Registry: Name Examination",
   "author": "Thor Wolpert <thor@wolpert.ca>",
   "private": true,
   "scripts": {
     "dev": "webpack-dev-server --inline --progress --config build/webpack.dev.conf.js",
     "start": "npm run dev",
-    "unit": "jest --config test/unit/jest.conf.js --runInBand --coverage --verbose",
-    "features": "jest --config test/unit/jest.conf.js test/features/specs/ --coverage --runInBand",
```

Comment:
Removed for the time being. Just a tidal wave of errors poor coverage. 
Will re implement when it makes sense.

Reply:
Agreed.